janitor/utils.py

```diff
@@ -405,3 +406,219 @@ def _wrapped(x, *args, **kwargs):
             return return_val
 
     return _wrapped
+
+
+def _check_instance(entry: Dict):
+    """
+    Function to check instances in the expand_grid function
+    This checks if entry is a dictionary,
+    checks the instance of value in key:value pairs in entry,
+    and makes changes to other types as deemed necessary. 
+    """
+    # entry has to be a dictionary
+    if not entry:
+        raise ValueError("passed dictionary cannot be empty")
+    # if it is a number or string, wrap in a list
```

Comment:
If I'm reading the code correctly, this should be:

```suggestion
    # if it is a NoneType, number, boolean, or string, then wrap in a list
```